src/modules/projects/components/tables/ProjectOutgoingReferralPostingsTable.tsx

```diff
@@ -0,0 +1,86 @@
+import RouterLink from '@/components/elements/RouterLink';
+import { ColumnDef } from '@/components/elements/table/types';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import HmisEnum from '@/modules/hmis/components/HmisEnum';
+import { parseAndFormatDate } from '@/modules/hmis/hmisUtil';
+import ReferralPostingStatusDisplay from '@/modules/referrals/components/ReferralPostingStatusDisplay';
+import { Routes } from '@/routes/routes';
+import { HmisEnums } from '@/types/gqlEnums';
+import {
+  GetProjectOutgoingReferralPostingsDocument,
+  GetProjectOutgoingReferralPostingsQuery,
+  GetProjectOutgoingReferralPostingsQueryVariables,
+  ReferralPostingFieldsFragment,
+} from '@/types/gqlTypes';
+import generateSafePath from '@/utils/generateSafePath';
+
+const columns: ColumnDef<ReferralPostingFieldsFragment>[] = [
+  {
+    header: 'Referral Date',
+    render: (row: ReferralPostingFieldsFragment) =>
+      parseAndFormatDate(row.referralDate),
+  },
+  {
+    header: 'Project Referred To',
+    linkTreatment: true,
+    render: ({ project }: ReferralPostingFieldsFragment) =>
+      project ? (
+        <>
+          <RouterLink
+            to={generateSafePath(Routes.PROJECT, {
+              projectId: project.id,
+            })}
```

Comment:
Let's just link directly over to the referral detail page in the receiving project. I think that is the most common use case for navigating to the other project would be to make a change to the status / see posting details.
```
            to={generateSafePath(ProjectDashboardRoutes.REFERRAL_POSTING, {
              projectId: project.id,
              referralPostingId: row.id,
            })}
```